src/main_thread/tracks_store/tracks_store.ts

```diff
@@ -1469,6 +1486,25 @@ function toExposedPeriod(p: IPeriodMetadata): IPeriod {
   return { start: p.start, end: p.end, id: p.id };
 }
 
+function findNextPlayableAdaptation(
```

Comment:
From that function's point of view, I'm under the impression that this is not really the "next" playable Adaptation, just the first playable adaptation, no?

Reply:
I have changed it.